src/test/regress/sql/seclabel.sql

```diff
@@ -0,0 +1,55 @@
+--
+-- SECLABEL
+--
+-- Test suite for SECURITY LABEL ON ROLE statements
+--
+
+-- first we remove one of the worker nodes to be able to test
+-- citus_add_node later
+SELECT citus_remove_node('localhost', :worker_2_port);
+
+CREATE ROLE user1;
+
+-- check an invalid label for our current dummy hook citus_test_object_relabel
+SECURITY LABEL ON ROLE user1 IS 'invalid_label';
+
+-- if we disable metadata_sync, the command will not be propagated
+SET citus.enable_metadata_sync TO off;
+SECURITY LABEL ON ROLE user1 IS 'citus_unclassified';
+SELECT node_type, result FROM get_citus_tests_label_provider_labels('user1');
+
+RESET citus.enable_metadata_sync;
+
```

Comment:
Instead of querying seclables one by one you can use below select to query at once 

SELECT result from run_command_on_all_nodes(
  $$
SELECT objtype, objname, provider, label FROM pg_seclabels;
  $$
) ORDER BY result;

Reply:
Yes, good idea, I have updated to Onur's suggestion: https://github.com/citusdata/citus/pull/7304#discussion_r1386587096